packages/docusaurus/src/commands/external.ts

```diff
@@ -5,14 +5,13 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import fs from 'fs-extra';
 import {loadContext} from '../server';
 import {initPlugins} from '../server/plugins/init';
 import type {CommanderStatic} from 'commander';
 
-export async function externalCommand(
-  cli: CommanderStatic,
-  siteDir: string,
-): Promise<void> {
+export async function externalCommand(cli: CommanderStatic): Promise<void> {
+  const siteDir = await fs.realpath('.');
```

Comment:
🤔 that looks different from the other cli commands 🤪 

But same as before so maybe a different issue